src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs

```diff
@@ -733,31 +748,65 @@ protected sealed override bool Receive(object message)
 
         private void FailChunkExecution(ChunkExecutionFailure message)
         {
+            _remainingOperations++;
             var cause = message.Cause;
-            Log.Error(cause, "Failed to execute chunk for {0} requests", message.Requests.Length);
-
-            foreach (var req in message.Requests)
+            Log.Error(cause, "An error occurred during event batch processing (chunkId: {0})", message.ChunkId);
+            
```

Comment:
I'd keep the number of requests (i.e., ` (chunkId: {0}) of {1} requests`)

Reply:
Done